modules/apps/headless/headless-admin-user/headless-admin-user-test/src/testIntegration/java/com/liferay/headless/admin/user/resource/v1_0/test/UserAccountResourceTest.java

```diff
@@ -1837,6 +1840,67 @@ private void _testGetUserAccountsPageWithCustomFields() throws Exception {
 			userAccount);
 	}
 
+	private void _testGetUserAccountWithMoreExternalReferenceCodes()
+		throws Exception {
+
+		User user = UserTestUtil.addUser();
+
+		_userLocalService.addRoleUser(_role.getRoleId(), user.getUserId());
+
+		_userLocalService.addOrganizationUser(
+			_organization.getOrganizationId(), user.getUserId());
+
+		_userLocalService.addUserGroupUser(
+			_userGroup.getUserGroupId(), user.getUserId());
+
+		_userLocalService.addGroupUser(
+			testGroup.getGroupId(), user.getUserId());
```

Comment:
Probably it's better to put these in alphabetical order like addGroupUser, then addOrgUser, AddRoleUser, addUserGroupUser.